src/base/components/FeedList.js

```diff
@@ -25,8 +32,83 @@ class FeedListPage extends React.Component {
         tspan.setAttribute('x', 90);
       }
     }, 3000)
-  }
 
+    let connection = new signalR.HubConnectionBuilder()
+    .configureLogging(signalR.LogLevel.Debug)
+  .withUrl(gconfig.gatewaysServiceUrl+'/devHub')
+  .build();
+      
+    connection.on("ReceiveMessage", (topic ,jdata) => {
+          console.log("ReceiveMessage:["+topic+"] "+ jdata);
+          
+          if(topic === "/iot/zolertia/replay"){
+            jdata =  NormailizeJSON(jdata);
+            var data = JSON.parse(jdata);
+            if(data.Command == "on" && data.Result == "Success"){
+              toast.success("Zolertia Successfully Turned ON!!!");
+            }
+            else if(data.Command == "off" && data.Result == "Success"){
+              toast.success("Zolertia Successfully Turned OFF!!!");
+            }
+            else if(data.Command == "SetTX" && data.Result == "Success"){
+              toast.success("Successfully Set "+data.Value+" to TX Level!!!");
+            }
+            else if (data.Command == "SetCh" && data.Result ==  "Success"){
+              toast.success("Successfully Set "+data.Value+" Channel!!!");
+            }
+            else if(data.Commnad == "SetTX" && data.Result == "Failed"){
+              toast.error("Failed to Set "+data.Value+" to TX Level!!!");
+            }
+            else if (data.Command == "SetCh" && data.Result ==  "Failed"){
+              toast.error("Failed to Set "+data.Value+" Channel!!!");
```

Comment:
"Something goes wrong. Channel remains the same"

Reply:
Done